### DISCUSSION TITLE
Input text different values for placeholder & wire:model

### DISCUSSION THREAD

**Discussion by cbhai** [livewire/livewire#3514] [1. Help] [answered]
May be I am missing something very basic here... please help

I have a input box in livewire component 

```
<input type="text" 
	placeholder="estimateWorkitems[{{ $index }}]['unit']"
        wire:model="estimateWorkitems.{{ $index }}.quantity" />
```

Question 1) I want to display column named 'unit' as placeholder. But bind this textbox to column quantity. Above code just shows quantity(no unit value in placeholder)
Question 2) is name="" inside input textbox required? what is it's use?

(estimateworkitems is array. quantity default value is zero.)

**Comment by ryatkins** [comment, marked as answer]
You’ll need to use blade syntax here:

="{{ $estimateWorkitems[$index]['unit'] }}“

**Reply by joshhanley** [reply]
@cbhai ryan is correct, for question 1, you need to change the placeholder to the above

`placeholder="{{ $estimateWorkitems[$index]['unit'] }}"`

For question two, from Livewire perspective, the "name" attribute is no longer needed on a html input. But I'm not sure whether it may still be needed for accessibility reasons or not.

Hope this helps!

**Reply by cbhai** [reply]
@ryatkins @joshhanley Thank you for help

Are you suggesting that this textbox cannot be livewire ?

If I keep textbox with just 
placeholder="{{ $estimateWorkitems[$index]['unit'] }}“ 
this does correctly show placeholder.
However if I also add 
        wire:model="estimateWorkitems.{{ $index }}.quantity" />
textbox displays quantity (ovewrites 'unit' placeholder)

**Reply by joshhanley** [reply]
@cbhai no it should work. I'm not sure how you got that idea 🙂 it was just the way you had used the blade echos wouldn't have worked properly, so needed to be amended.

This new code looks good. Is it not working the way you are hoping? If so how are you expecting it to work?

**Reply by cbhai** [reply]
For above code if I keep quantity field null, it shows placeholder correctly (unit value). If I initialize quantity to 0 in mount, it overwrites placeholder. So I think that is right thing, may be that is correct behavior. Thank you once again
